File: Challenges/21 Days DSA Challenge/Gennady and his explosion string.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        /*
         * Your class should be named Main.
         * Read input as specified in the question.
         * Print output as specified in the question.
         */

        // Write your code here
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        String explosiveString = sc.next();

        while (str.length() > 0) {
            int index = str.indexOf(explosiveString);
            if (index == -1) {
                System.out.println(str);
                return;
            }
            str = str.substring(0, index) + str.substring(index + explosiveString.length());
        }
        System.out.println("FRULA");
    }
}

File: Challenges/21 Days DSA Challenge/Gennady and his explosion string.py
def main():
    str_input = input()
    explosive_string = input()

    while len(str_input) > 0:
        index = str_input.find(explosive_string)
        if index == -1:
            print(str_input)
            return
        str_input = str_input[:index] + \
            str_input[index + len(explosive_string):]

    print("FRULA")


if __name__ == "__main__":
    main()
